x-pack/plugins/maps/public/connected_components/mb_map/draw_control/draw_control.tsx

```diff
@@ -139,16 +140,18 @@ export class DrawControl extends Component<Props> {
         },
       });
       this._mbDrawControlAdded = true;
-      this.props.mbMap.getCanvas().style.cursor = 'crosshair';
       this.props.mbMap.on('draw.modechange', this._onModeChange);
       this.props.mbMap.on('draw.create', this._onDraw);
+
       if (this.props.onClick) {
         this.props.mbMap.on('click', this._onClick);
       }
     }
+    this.props.mbMap.getCanvas().style.cursor = 'crosshair';
```

Comment:
Lets clean this logic up so curser is only getting set in a single location. Right now, the curser is set to crosshair and then later in the if else block its reset to empty. Lets just change the logic to 

```
this.props.mbMap.getCanvas().style.cursor = !this.props.drawShape || this.props.drawShape === DRAW_SHAPE.SIMPLE_SELECT
  ? ''
  : 'curser';
```

Then remove setting cursor to empty in line 169